components/ColorDetails.js

```diff
@@ -27,12 +27,31 @@ export const ColorDetail = ({ color }) => {
     }
   });
   const fullColor = new Color(color);
+
+  const result = fullColor.tohwb(); // Call the original function
```

Comment:
@mtalhabaig3 thanks for the PR.
I think it is better to move this logic to a fuction and directly call it so that all these variables are local to a function instead of being part of the full scope. Also instead of using result as a variable name we can use something specific like hwb and formatedHwb.

Reply:
Alright, let me do it!